android/src/main/java/com/reactnativecommunity/webview/RNCWebChromeClient.java

```diff
@@ -85,6 +89,24 @@ public RNCWebChromeClient(RNCWebView webView) {
     public boolean onCreateWindow(WebView view, boolean isDialog, boolean isUserGesture, Message resultMsg) {
 
         final WebView newWebView = new WebView(view.getContext());
+
+        if(mHasOnOpenWindowEvent) {
+            newWebView.setWebViewClient(new WebViewClient(){
+            @Override
+            public boolean shouldOverrideUrlLoading (WebView subview, String url) {
+                WritableMap event = Arguments.createMap();
+                event.putString("targetUrl", url);
+
+                ((RNCWebView) view).dispatchEvent(
+                view,
+                new TopOpenWindowEvent(view.getId(), event)
+                );
```

Comment:
small nit: indentation
```suggestion
                ((RNCWebView) view).dispatchEvent(
                    view,
                    new TopOpenWindowEvent(view.getId(), event)
                );
```

I hadn't noticed until now that the indentation changed from 2-space to 4-space with the new architecture refactor... not certain if that was intentional... but might as well be consistent in this case.

Reply:
Fixed in: https://github.com/react-native-webview/react-native-webview/compare/5a835821371ba93f5a5ac120c88097fc3c918ff4..610aee3309dfd69739c28a5bb4579b91015d0394